artemis-server/src/main/java/org/apache/activemq/artemis/core/deployers/impl/FileConfigurationParser.java

```diff
@@ -213,6 +213,8 @@
 
    private static final String MAX_SIZE_BYTES_NODE_NAME = "max-size-bytes";
 
+   private static final String MAX_MESSAGES_NODE_NAME = "max-messages";
```

Comment:
I wonder if `max-size-messages` would be better here for consistency with `max-size-bytes`.

Reply:
I thought about that..


I thought that number of messages would not be a size thing... (quantity versus size)..

If you think the term size would be okay here I can definitely include it.

Reply:
@jbertram ok.. I like that actually.. I had thought about max-size-message and I thought it wouldn't be a good term because of English grammar...

although global-max-messages it sounds fine as is I think? 

global-max-size-messages wouldn't sound weird?

Reply:
Yes, I think `global-max-messages` is the best choice there.